app/db/models/user.js

```diff
@@ -23,7 +29,16 @@ module.exports = (sequelize, DataTypes) => {
         field: 'auth0_id'
       },
       email: { type: DataTypes.STRING, unique: true },
-      roles: DataTypes.ARRAY(DataTypes.TEXT),
+      roles: {
+        type: DataTypes.ARRAY(DataTypes.TEXT),
+        defaultValue: ['USER'],
```

Comment:
see https://sequelize.org/master/manual/validations-and-constraints.html

Does a migration need to be written for adding a validator? I really can't say, I can't test it to see if its there at the moment, and the community and documentation are all over the place